indigo_app/views/base.py

```diff
@@ -6,6 +9,16 @@
 from indigo_api.models import Country
 
 
+def page_count(works):
+    page_count = 0
+    for w in works:
+        for d in w.document_set.undeleted():
+            xml = etree.fromstring(d.document_xml)
+            text = '\n'.join(x.strip() for x in xml.xpath('//a:*//text()', namespaces={'a': xml.nsmap[None]}))
+            page_count += ceil(len(text.split()) / 250)
```

Comment:
i'm not following the logic here -- why join with newlines?

Are you estimating 250 words per page?

Reply:
Joining with newlines was easier for debugging when I was checking it was doing the right thing; updating to be a little simpler.

And yes, 250 words per page.